modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -35,15 +36,14 @@ var backendServiceInvoker = function () {
      * This methoad reads the token pair from the session and return the access token.
      * If the token pair s not set in the session this will send a redirect to the login page.
      */
-    function getAccessToken() {
+    privateMethods.getAccessToken = function () {
         var tokenPair = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER);
         if (tokenPair) {
             return tokenPair.accessToken;
         } else {
             response.sendRedirect(mdmProps["httpsURL"] + "/emm/login");
         }
-
-    }
+    };
 
```

Comment:
it's not appropriate to use a sendRedirect() here in a function meant to get an access token.